module/documents/activity/mixin.mjs

```diff
@@ -593,7 +593,7 @@ export default Base => class extends PseudoDocumentMixin(Base) {
    * @param {Item5e} item                                 Clone of the item that contains this activity.
    * @protected
    */
-  _prepareUsageScaling(usageConfig, messageConfig, item) {
+  async _prepareUsageScaling(usageConfig, messageConfig, item) {
```

Comment:
Noticed that `EnchantActivity` overrides this and makes it `async`, so I updated the signature to match, and made sure it was appropriately `await`ed in `Activity#use`.